main.py

```diff
@@ -71,6 +71,8 @@
 # The publisher object for the queue with patterns matching objects updated by
 # loading submissions.
 _UPDATED_QUEUE_PUBLISHER = None
+# Storage bucket name
```

Comment:
```suggestion
# KCIDB cache storage bucket name
```

Based on the description we've used on _CACHE_CLIENT

Reply:
okay